src/Store.js

```diff
@@ -595,7 +595,7 @@ class Store {
       throw new Error(`Could not find a model for '${type}'`)
     }
 
-    return new ModelKlass({ id, store, relationships, ...attributes })
+    return new ModelKlass({ store, relationships, ...attributes, id: id.toString() })
```

Comment:
This is a good idea 👍  We should do the same thing on the api side. We aren't consistent over there either.

Reply:
what are your thoughts around this? passing back all our ids as strings from the API? I feel like most of the problems we're going to run into with this change will be related to `===` comparisons between numbers and strings, so if everything is a string from the source we hopefully won't have change those occurrences.

Reply:
> passing back all our ids as strings from the API?

Yes, but we should check here as well, since we may instantiate on the client with values which didn't originate from the api.

>  I feel like most of the problems we're going to run into with this change will be related to === comparisons between numbers and strings, so if everything is a string from the source we hopefully won't have change those occurrences.

I agree. I think we're on the same page, but maybe I'm misunderstanding your comment. Lmk if you want to talk about it.

Reply:
I just had a talked with Joe about it. First pass I think the plan will be to prefer ids on relationships rather than a direct relationship id (`stage.organization.id` vs `stage.organization_id`) but also make sure any places where the relationship is useful to declare it as a String vs Number in the JS model. We can then take steps on the serializable front to actually pass those values as strings, but this'll help us take it in steps and the priority is that the JS code knows what types to work with.